app/src/androidTest/java/net/aiscope/gdd_app/ui/metadata/SampleImagesAdapterTest.kt

```diff
@@ -0,0 +1,69 @@
+package net.aiscope.gdd_app.ui.metadata
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.File
+import java.io.InputStream
+
+@RunWith(AndroidJUnit4::class)
+class SampleImagesAdapterTest {

```

Comment:
The tests in this class don't really have to do anything with the adapter itself, but with some helper method that happens to be in the same file as the adapter. Thus, I am thinking, maybe it would make sense to move that method to its own file? Honestly, I wouldn't mind having it in the `BitmapReader` class, but whatever you prefer :)

Reply:
I had it in BitmapReader initially but moved it back for a reason that now does not exist anymore 😆 

Probs best to move it back

Reply:
Moved